spec/css/plain/import/conditions.hrx

```diff
@@ -149,6 +181,30 @@ $a: "a";
 <===> media/complex/output.css
 @import url("a.css") handheld and (max-width: 400px);
 
+<===>
+================================================================================
+<===> media/list/after_ident/input.scss
+@import "a" b, (c: d) and (e: f), g;
+
+<===> media/list/after_ident/output.css
+@import "a" b, (c: d) and (e: f), g;
+
+<===>
+================================================================================
+<===> media/list/after_feature/input.scss
+@import "a" (b: c), (d: e) and (f: g), h;
```

Comment:
this looks wrong to me. The `,` separates several ImportArgument, and `(d: e)` is not a valid ImportUrl

Reply:
This is valid CSS! An `@import` [may be followed by a `<media-query-list>`](https://www.w3.org/TR/css-cascade-5/#at-import), and `(b: c), (d: e) and (f: g), h` is [a valid `<media-query-list>`](https://www.w3.org/TR/mediaqueries-4/#typedef-media-query-list).

Reply:
The the spec is ambiguous when allowing multiple ImportArgument